snippets/powershell/Post-NB-Cleanup-Assets.ps1

```diff
@@ -0,0 +1,168 @@
+<#
+
+.SYNOPSIS
+This sample script demonstrates the use of NetBackup REST API for listing all assets.
+
+.DESCRIPTION
+This script will get a list of asset in the AssetDB that matches the data filter.
+
+.EXAMPLE
+./Get-NB-Assets.ps1 -nbmaster "nb-master.example.com" -username "administrator" -password "secret" -filter "workloadType eq 'VMware'"
```

Comment:
Script is Post-NB-Cleanup-Assets. Example lists something else.

Reply:
Should the usage mention cleanupTime as well?

Reply:
Yes. I fixed.